### PR TITLE
Conversation page logic

### PR DESCRIPTION
**Pull request by achen443**
Implemented logic for Conversation page using Sohum's API

Navigate to conversations tab, this will pull up all your recent conversations with the most recent being at the top.
@anand-ts will do the styling for this page

### CODE REVIEW

**Review by sohumraoGT — APPROVED** [review]

**Review by anand-ts — APPROVED** [review]
Thanks @achen443, looks great - will push a PR for the stylist soon.

**Review by clarkcousins — APPROVED** [review]
no conversations yet of course but it tells me there are none. good work!

File: beauty-consulting-app/app/assets/components/ConversationList.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		<ConversationItem
+			name={item.name}
+			messagePreview={item.messagePreview}
+			timestamp={item.timestamp}
+		/>
+	);
+
+	return (
+		<View style={styles.container}>
+			{conversations && conversations.length > 0 ? (
```
Comment (clarkcousins) [inline beauty-consulting-app/app/assets/components/ConversationList.js:16]: && may be causing "Warning: Text strings must be rendered within a <Text> component

**Comment by DavidHe137** [comment]
Changes in merged in #121.

**Pull request closed without merging**